internal/profiler/profiler.go

```diff
@@ -21,3 +21,17 @@ func Init(svcName, version string, blockProfileRate int) {
 		}
 	}
 }
+
+// InitLightweight starts the supported profilers IFF the environment variable is set.
+// Compared to Init, it disables mutex profiling and forced GC to reduce its overhead.
```

Comment:
```suggestion
// Compared to Init, it disables mutex profiling and forces GC to reduce its overhead.
```

Reply:
I think Julies comment is the more correct. It just might be confusing since I think the meaning is `disables (mutex profiling AND forced GC)`